2009-08-09  Aurelien Jarno  <aurelien@aurel32.net>

	* Makeconfig: convert i586 and i686 into i486.

2005-01-20  Tollef Fog Heen  <tfheen@debian.org> 

	* Makeconfig: add multiarch support

--- a/Makeconfig
+++ b/Makeconfig
@@ -191,6 +191,13 @@
 endif
 inst_libdir = $(install_root)$(libdir)
 
+# Multiarch places to look for libraries
+ifndef extra_libdir
+multiarch-machine = $(subst i586,i486,$(config-machine))
+multiarch-machine = $(subst i686,i486,$(multiarch-machine))
+extra_libdir = /lib/$(multiarch-machine)-$(config-os):$(exec_prefix)/lib/$(multiarch-machine)-$(config-os)
+endif
+
 # Where to install the shared library and dynamic linker.
 ifndef slibdir
 slibdir = $(exec_prefix)/lib
@@ -501,6 +508,10 @@
 default-rpath = $(libdir)
 endif
 
+ifdef extra_libdir
+default-rpath += :$(extra_libdir)
+endif
+
 ifndef link-extra-libs
 link-extra-libs = $(LDLIBS-$(@F))
 link-extra-libs-static = $(link-extra-libs)
